jduong09/thirteen
# AI Logic : Handling All Card Combinations

Partially resolves #22 

## Changes/Additions
- Added logic so AI can be given current turn combo (single/high card | pair | three of a kind | four of a kind | sequence | double sequence)
- If AI has multiple possible plays for a turn combo, AI will choose lowest value combo that beats the middle pile value

## Hot To Test
- Checkout to `issue-22--ai-logic` and pull changes with `git pull`
- Navigate to `localhost:3000` in your browser
- Shuffle deck, the first turn will be user. Choose combination type you want to test and play valid turn.
- CPU will try to play valid turn and pass if there is no valid turn.
- Use console logs to verify correct turn and follow along with middle pile as well.
- When cycle ends, refresh browser to continue testing.
